pyproject.toml

```diff
@@ -0,0 +1,17 @@
+[tool.black]
+line-lenghth = 119
```

Comment:
commited this by error on you fork while testing github ide... , just to let you know.

Reply:
no problem!